docs/developers/localdevelscratch.md

```diff
@@ -366,16 +358,16 @@ Now you should be able to access the sites on [http://hypha.test/](http://hypha.
 
 ### Run tests
 
-Hypha has specific settings for testing so specify them when you run the "test" command.
+Hypha uses `py.test` test runner and settings from `hypha/settings/testing.py` to run its tests. Run the test with:
 
 ```text
-$ python manage.py test --settings=hypha.settings.test
+$ make test
 ```
 
 If you need to rerun the tests several times this will speed them up considerably.
 
 ```text
-$ python manage.py test --parallel --keepdb --settings=hypha.settings.test
+$ py.test --reuse-db
```

Comment:
Looks like `make test` runs `make py-test` and that command always use "--reuse-db".

Reply:
And I think it is recommended to use `pytest` instead of the old `py.test`.

Reply:
I'll look at them, and update. :)

Reply:
Update the text to be more descriptive. Thanks for the feedback.